delivery_schenker/models/delivery_carrier.py

```diff
@@ -358,12 +358,27 @@ def _schenker_shipping_information_package_volume(self, picking, package):
             ]
         )
 
+    def _schenker_float_round(self, value, precision_digits):
+        """
+        The schenker api requires 2 decimal points.
+        Round up to avoid 0 if <0.005.
+        round -> repr -> float to avoid extra decimals that error with the Schema.
+        float_round(2.086, precision_digits=2) = 2.0100000000000002
+        """
+        return float(
```

Comment:
Why reconvert back to float? You will get back extra decimals:
```
print('{0:.16f}'.format(float("2.09")))
2.0899999999999999

```
SOAP is transmitting in XML, so converting it all to text, so once you have the representation with 2 decimals, you better send that, you don't need a real float in your dict.